tests/e2e/notes.test.ts

```diff
@@ -0,0 +1,74 @@
+import { faker } from '@faker-js/faker'
+import { expect, test } from '../playwright-utils.ts'
+
+test('Users can create notes', async ({ login, page }) => {
+	const user = await login()
+	await page.goto(`/users/${user.username}/notes`)
+
+	const newNote = createNewNote()
+	await page.locator("//a[text()='+ New Note']").click()
+
+	// blank form submission should result in errors
+	await page.locator('button[type="submit"]').click()
+	await expect(page.locator('ul#note-editor-title-error')).toBeVisible()
+	await expect(page.locator('ul#note-editor-content-error')).toBeVisible()
+
+	// fill in form and submit
+	await page.locator('input[name="title"]').fill(newNote.title)
+	await page.locator('textarea[name="content"]').fill(newNote.content)
+	await page.locator('button[type="submit"]').click()
+	await expect(page).toHaveURL(new RegExp(`/users/${user.username}/notes/*`))
```

Comment:
Should this be:

```suggestion
	await expect(page).toHaveURL(new RegExp(`/users/${user.username}/notes/.*`))
```

Same thing below